app/services/billable_metrics/pay_in_advance_aggregation_service.rb

```diff
@@ -2,8 +2,8 @@
 
 module BillableMetrics
```

Comment:
Since the main argument is now a charge, shoul'nt we change the module to `Charges` for this one?

Reply:
👍